statscache/producer.py

```diff
@@ -12,95 +11,35 @@
 log = logging.getLogger("fedmsg")
 
 
-class StatsProducerBase(moksha.hub.api.PollingProducer):
+class StatsProducer(moksha.hub.api.PollingProducer):
     """
     An abstract base class for dynamically generated producers. Subclasses need
```

Comment:
It seems this docstring is out of date, or am I misunderstanding things?  There is now only one polling producer in play, so there are no more dynamically generated producers.